readme.md

```diff
@@ -235,7 +235,7 @@ Add the following secrets:
 
 ```
 # macOS
-gpg --armor --export-secret-keys $LONG_ID | base64 | pbcopy
+gpg --armor --export-secret-keys $LONG_ID | base64 -w0 | pbcopy
```

Comment:
Are you using GNU Coreutils? Quick Googling says macOS version of base64 doesn't support `-w` flag. I can confirm this locally on macOS as well.

Reply:
Looks like I'm using coreutils.

```
$ which base64
/usr/local/opt/coreutils/libexec/gnubin/base64
```